packages/mendel-config/src/bundle-config.js

```diff
@@ -19,10 +33,26 @@ function BundleConfig(id, options, config) {
     this.external = flattenArrays(options.external || []);
     this.outlet = options.outlet;
 
+    console.log(options, this.options);
+
     this.options = without(
         options,
         ['generator', 'outfile', 'outlet', 'entries', 'require', 'external']
     );
+
+    Object.keys(this.options).forEach((optionKey) => {
+        if (supportedOptions.indexOf(optionKey) === -1) {
+            unsupportedOptions.push(optionKey);
+        }
+    });
+
+    if (unsupportedOptions.length > 0) {
+        debug([
+            '[Config][WARN] Following options are not supported for bundle: ' + id + '.\n',
```

Comment:
If you try out `DEBUG=mendel* mendel` the output already says in color that it is part of "mendel:config". So I think it might be better to remove the `[Config]` portion of the string.